app/src/main/java/ch/hikemate/app/ui/profile/EditProfileScreen.kt

```diff
@@ -0,0 +1,174 @@
+  }
+
+  Column(
+      modifier =
+          Modifier.testTag(Screen.EDIT_PROFILE)
+              .padding(
+                  // Add padding to the sidebar padding
+                  start = 16.dp,
+                  end = 16.dp,
+                  top = 16.dp,
+              ),
+      verticalArrangement = Arrangement.spacedBy(16.dp)) {
+        BackButton(navigationActions)
+        Text(
+            context.getString(R.string.edit_profile_screen_title),
+            style = TextStyle(fontWeight = FontWeight.Bold, fontSize = 32.sp),
+            modifier = Modifier.testTag(EditProfileScreen.TEST_TAG_TITLE))
+
+        OutlinedTextField(
+            modifier = Modifier.fillMaxWidth().testTag(EditProfileScreen.TEST_TAG_NAME_INPUT),
+            colors =
+                OutlinedTextFieldDefaults.colors()
+                    .copy(
+                        focusedLabelColor = primaryColor,
+                        focusedIndicatorColor = primaryColor,
+                        cursorColor = primaryColor,
+                        textSelectionColors =
+                            TextSelectionColors(
+                                handleColor = primaryColor,
+                                backgroundColor = primaryColor,
+                            )),
+            value = name,
+            onValueChange = { name = it },
+            label = { Text(context.getString(R.string.profile_screen_name_label)) })
+
+        Column(
+            verticalArrangement = Arrangement.spacedBy(2.dp),
+        ) {
+          Text(
+              context.getString(R.string.profile_screen_hiking_level_label),
+              style = TextStyle(fontWeight = FontWeight.Bold, fontSize = 20.sp),
+              modifier = Modifier.testTag(EditProfileScreen.TEST_TAG_HIKING_LEVEL_LABEL))
+          SingleChoiceSegmentedButtonRow {
+            HikingLevel.values().forEachIndexed { index, fitLevel ->
+              SegmentedButton(
+                  modifier =
+                      Modifier.testTag(
+                          when (fitLevel) {
+                            HikingLevel.BEGINNER ->
+                                EditProfileScreen.TEST_TAG_HIKING_LEVEL_CHOICE_BEGINNER
+                            HikingLevel.INTERMEDIATE ->
+                                EditProfileScreen.TEST_TAG_HIKING_LEVEL_CHOICE_INTERMEDIATE
+                            HikingLevel.EXPERT ->
+                                EditProfileScreen.TEST_TAG_HIKING_LEVEL_CHOICE_EXPERT
+                          }),
+                  shape =
+                      SegmentedButtonDefaults.itemShape(
+                          index = index, count = HikingLevel.values().size),
+                  colors =
+                      SegmentedButtonDefaults.colors()
+                          .copy(
+                              activeContainerColor = primaryColor,
+                              activeContentColor = Color.White,
```

Comment:
Are you talking about the `Color.White` ?

Reply:
Yep the white color !

Reply:
Can you elaborate on the purpose of this color then ?
In this context, the color is set to white in order to see the text when a level is selected. It is directly related to the `activeContainerColor` being blue.

`lightColor`doesn't seem to be very explicit here and I don't really know how should I name a such specific color. Can you help me ?

Reply:
This is just an idea for improvement by centralizing all colors used in components in a single file to make it easier to modify. Putting "Color.White" is a hard coded value, and I think we should find a way to avoid them

Reply:
Maybe you can name it something like "primaryBackgroundColor" ?

Reply:
I think we will do that in another PR as this should be a project wide correction. Do you agree ?

Reply:
Yup